src/java/org/apache/nutch/service/resources/SeedResource.java

```diff
@@ -0,0 +1,112 @@
+
+
+/*******************************************************************************
```

Comment:
please don't invent new ways to show the header. Just use the way the other files have a header. Please see them.